core/src/display_object/edit_text.rs

```diff
@@ -197,6 +197,13 @@ impl<'gc> EditText<'gc> {
     // This seems to be OS-independent
     const INPUT_NEWLINE: char = '\r';
 
+    /// Gutter is the constant internal padding of a text field.
+    /// It applies to each side and cannot be changed.
+    ///
+    /// See <https://open-flash.github.io/mirrors/as2-language-reference/TextFormat.html#getTextExtent()>.
+    /// See <https://help.adobe.com/en_US/FlashPlatform/reference/actionscript/3/flash/text/TextLineMetrics.html>.
+    const GUTTER: Twips = Twips::new(40);
```

Comment:
I think using `Twip::from_pixels_i32` would have been slightly more obvious.

Reply:
Whoops, merged just a second before this comment :(